src/main/webapp/WEB-INF/jsp/editNews.jsp

```diff
@@ -22,13 +22,8 @@
 <jsp:directive.include file="/WEB-INF/jsp/include.jsp"/>
 <c:set var="n"><portlet:namespace/></c:set>
 
-<style type="text/css">
-	table.edit-news { width: 100%; }
-	table.edit-news td { font-size: 1.1em; }
-	table.edit-news td.instruction { color: #666; font-size: 1em; text-align: center; vertical-align: bottom; }
-</style>
+<link href="<c:url value="/css/newsreader.css"/>" rel="stylesheet" type="text/css" />
 
-<jsp:directive.include file="/WEB-INF/jsp/scripts.jsp"/>
```

Comment:
Yes, if this line existed before, it should remain. I may have removed it in haste.